jwt/jwt_test.go

```diff
@@ -87,3 +88,26 @@ func TestVerify_BadClaims(t *testing.T) {
 	assert.Error(t, err)
 	assert.False(t, verified, "expected !verified")
 }
+
+func TestDecodeJWTClaims(t *testing.T) {
+	did, err := didjwk.Create()
+	assert.NoError(t, err)
+
+	nonce := "abcd123"
+	claims := jwt.Claims{
+		Issuer: did.ID,
+		Misc:   map[string]interface{}{"c_nonce": nonce},
```

Comment:
`interface{}` still works, but there's an alias for it now which is much more convenient: `any`

Reply:
IMO I prefer the language-native way of `interface` ... less convenient but more clear (at least to newbies)

Reply:
oh nice @alecthomas i had no idea about the alias